components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/inboundfactory/InboundRequestProcessorFactoryImpl.java

```diff
@@ -61,9 +60,6 @@ public InboundRequestProcessor createInboundProcessor(InboundProcessorParams par
                 case jms:
                     inboundRequestProcessor = new JMSProcessor(params);
                     break;
-                case file:
```

Comment:
why?

Reply:
because we are removing Protocol/File in this PR